twitter_scraper/modules/profile.py

```diff
@@ -66,6 +67,9 @@ def __parse_profile(self, page):
         if not self.location:
             self.location = None
 
+        self.joined_date = html.find(".ProfileHeaderCard-joinDateText")[0].text.replace('Joined ', '')
```

Comment:
Oh, my bad. We are not working on dictionaries here. You can not use .get method. You need to check if **html.find(".ProfileHeaderCard-joinDateText")** exists. If exists everything good, you can get the first element and parse joined date. But if not exists self.joined_date must be equal to None